SearchRepos is failing because pandas append went away

```
Prepare all required actions
Run ./.github/actions/single_search
Run search_github --gh_token *** --start_date [2](https://github.com/CERTCC/labyrinth/actions/runs/5179586538/jobs/9332662297#step:6:2)023-06-04 --end_date 2023-06-05  attack poc
Starting 1 queries
Search: attack poc pushed:2023-06-04..2023-06-05
Found 3 results for attack poc pushed:2023-06-04..2023-06-05
df has 3 rows
df has 3 rows after dropna
df has 3 rows after drop out of range dates
Search found 3 results for 2023-06-05
Read 1 records from results/2023/06/05/2023-06-05_attack_poc.json.
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.16/x64/bin/search_github", line 202, in <module>
    main(search_str, args.start_date, args.end_date, args.overwrite)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/bin/search_github", line 139, in main
    out_df = json_df.append(new_df, ignore_index=True)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/pandas/core/generic.py", line 5989, in __getattr__
    return object.__getattribute__(self, name)
AttributeError: 'DataFrame' object has no attribute 'append'
Error: Process completed with exit code 1.
```

--- a/scripts/search_github
+++ b/scripts/search_github
@@ -136,7 +136,7 @@ def main(query, start_date=None, end_date=None, overwrite=False):
 
             if len(new_df):
                 # append new rows to existing
-                out_df = json_df.append(new_df, ignore_index=True)
+                out_df = pd.concat(json_df,new_df)
             else:
                 # we can skip to the next day if new_df is empty
                 print(f"No new results for {dt_pfx}, skipping ahead", flush=True)

